clients/src/test/java/org/apache/kafka/common/security/oauthbearer/internals/OAuthBearerClientInitialResponseTest.java

```diff
@@ -59,7 +86,5 @@ public void testRfc7688Example() throws Exception {
         OAuthBearerClientInitialResponse response = new OAuthBearerClientInitialResponse(message.getBytes(StandardCharsets.UTF_8));
         assertEquals("vF9dft4qmTc2Nvb3RlckBhbHRhdmlzdGEuY29tCg", response.tokenValue());
         assertEquals("user@example.com", response.authorizationId());
-        assertEquals("server.example.com", response.propertyValue("host"));
```

Comment:
Why delete these 2 lines?  Can you just call `response.extensions().map().get()` instead of `response.propertyValue()`?

Reply:
Yes. My bad, sorry